setup.cfg

```diff
@@ -60,7 +61,7 @@ image =
   scipy>=1.7.0,!=1.10.0
 jpeg2000 =
   glymur>=0.9.1,!=0.9.5
-  lxml>=4.8.0
+  lxml>=4.8.0,!=5.0.0
```

Comment:
Is it better to pin this to `<5` for now? Or is the issue just with exactly `5.0.0`?

Reply:
We can remove this pin now, they uploaded the wheels needed.

Reply:
turns out not to be the case, we need to wait for lxml 5.0.1.

5.0.1 (2024-??-??)
Bugs fixed
LP#2046208: Parsing non-BMP Python Unicode strings could fail on macOS.

LP#2044225: When incrementally parsing broken HTML, reporting start events on
missing structural tags failed and could lead to subsequent exceptions.

LP#2045435: Some (not all) issues with stricter C compilers were resolved.

The binary wheels in the 5.0.0 release did not validate cleanly (but installed ok).